pkg/auth/handler/webapp/viewmodels/settings_profile.go

```diff
@@ -92,7 +93,7 @@ type SettingsProfileViewModeler struct {
 
 // nolint: gocognit
 func (m *SettingsProfileViewModeler) ViewModel(userID string) (*SettingsProfileViewModel, error) {
-	var emails []string
+	var emails setutil.Set[string]
```

Comment:
I guess we need to apply the same logic to phone numbers and usernames?

Reply:
Updated, thanks!